src/mmw/apps/modeling/geoprocessing.py

```diff
@@ -47,7 +48,27 @@
 }
 
 
-def histogram(polygons):
+def histogramPoll(jobId):
+    host = settings.GEOP['host']
+    port = settings.GEOP['port']
+    path = '/jobs/' + jobId
+    conn = httplib.HTTPConnection("%s:%s" % (host, port))
```

Comment:
I don't have a bunch of experience with `requests`, but I am in favor of leaning on some of the functionality it provides here. If it isn't already a dependency somewhere, I think adding it for this use case is valid.